packages/alfa-rules/src/sia-r23/rule.ts

```diff
@@ -25,53 +25,55 @@ export default Rule.Atomic.of<Page, Element, Question>({
 
       expectations(target) {
         return {
-          1: Question.of(
-            "transcript",
-            "node",
-            target,
-            `Where is the transcript of the \`<audio>\` element?`
-          ).map((transcript) => {
-            if (transcript.isNone()) {
-              return Question.of(
-                "transcript-link",
-                "node",
-                target,
-                `Where is the link pointing to the transcript of the \`<audio>\`
+          1: <Interview<Question, Element, Option<Result<Diagnostic>>>>(
```

Comment:
I don't understand why the typing information is needed for some of the rules but not for all (looks like not all semi-automatic rules are here).

Reply:
It's only needed for the cases where multiple different types of questions are nested as the outer question seems to dictate the inference of the rest.

Reply:
This actually makes some sort of sense…